packages/gitmojis/src/gitmojis.json

```
@@ -1,4 +1,5 @@
 {
+  "$schema": "https://raw.githubusercontent.com/carloscuesta/gitmoji/master/packages/gitmojis/src/schema.json",
```

I think we should probably expose this as a static file through an API endpoint / route rewrite to avoid coupling the "file system" into the `json` file

```suggestion
  "$schema": "https://gitmoji.dev/api/gitmojis/schema",
```

---

yeah, that would be a great idea :sweat_smile:

---

I can help you with that if you don't have the time, should I go ahead? 🖖🏼

---

sure, if you are free, you can do it, tho I might have time to do ti tomorrow :D